utils/common/src/main/kotlin/Utils.kt

```diff
@@ -26,13 +26,14 @@ import java.util.EnumSet
 
 /**
  * A list of directories used by version control systems to store metadata.
+ * The list covers also version control systems not supported by ORT, because such directories
+ * should never be scanned.
```

Comment:
`VCS_DIRECTORIES` are not only ignored by the scanner, but also skipped by the analyzer, so we should generalize this e.g. by saying "... never be *considered*."